doc/source/metadata/Principles.rst

```diff
@@ -0,0 +1,208 @@
+Principles
+==========
+
+.. contents:: Contents
+    :depth: 3
+
+Overview
+--------
+
+The principles applied to the metadata within IMPROVER have been listed
+in the Introduction, but are discussed in more detail in this section.
+
+
+Conformance to the CF Metadata Conventions
+------------------------------------------
+
+As far as possible the use of metadata within IMPROVER should conform to the
+`CF Metadata Conventions`_ (CF),
+but recognising the need to extend this in some areas.
+
+However, whilst CF does a good job with 'where' and 'what' metadata
+(see note below), it does not really cover 'who' metadata.
+Therefore, it is helpful to keep CF (and direct extensions to this)
+focussed on these aspects and not try to stretch it to cover 'who'.
+
+An example is CF cell methods, which are intended to cover 'what'
+(i.e. to ensure that data can be correctly interpreted) 
+but can unwittingly be extended into 'who'
+(e.g. to try to describe the post-processing that have been applied), 
+which can complicate its main purpose. 
+
+.. note::
+
+    **Metadata Categorization** 
+
+    To inform decisions about the implementation of metadata, 
+    it can be helpful to consider three types of metadata in terms
+    of what they are trying to achieve:
+
+    * **Where** - location in space and time,
+      and additional 'data space' information
+
+    * **What** - what is this value?
+      i.e. phenomenon, units, extra characteristics
+
+    * **Who** - data identity,
+      i.e. where it came from, provenance, data processing, etc
+
+Clear purpose
+-------------
+
+Metadata should not be present, if it does not have a purpose
+(e.g. it should be included just because it might be useful in future), as,
+if no one is using it, it is unlikely to be properly maintained.
```

Comment:
I thought that this might be clearer.

```suggestion
Metadata should only be present if it has a purpose. It should not 
be included just because it might be useful in future, as if no one is 
using it, it is unlikely to be properly maintained.
```

Reply:
Agreed - making change.